extras/cyphal++/examples/example-01-opencyphal-basic-node/README.md

```diff
@@ -0,0 +1,34 @@
+<a href="https://opencyphal.org/"><img align="right" src="https://raw.githubusercontent.com/107-systems/.github/main/logo/opencyphal.svg" width="25%"></a>
+:floppy_disk: `example-01-opencyphal-basic-node`
+================================================
+* Setup a virtual CAN interface (`vcan0`)
+```bash
+sudo . setup_vcan.sh
```

Comment:
```suggestion
sudo ./setup_vcan.sh
```

Reply:
Actually `. setup_vcan.sh` would be enough, due to `[ "$UID" -eq 0 ] || exec sudo bash "$0" "$@"`